test/integration/cli/cli-test.coffee

```diff
@@ -1,5 +1,7 @@
-{assert} = require('chai')
 net = require('net')
+path = require('path')
+
+{assert} = require('chai')
```

Comment:
Just a habit - I like to differentiate between default and destructured imports (requires).